girder/api/rest.py

```diff
@@ -252,6 +252,45 @@ def setCurrentUser(user):
     cherrypy.request.girderUser = user
 
 
+def setContentDisposition(filename, disposition='attachment', setHeader=True):
+    """
+    Set the content disposition header to either inline or attachment, and
+    specify a filename that is properly escaped.
```

Comment:
Can you include a reference to the RFC that this should confirm to?

Also, did you review the relevant RFC, and how well does this implementation conform to it?

Reply:
As often the MDN page has a good summary of the issue and links to the relevant RFCs (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition), and see https://tools.ietf.org/html/rfc2183 and https://tools.ietf.org/html/rfc6266.

I'll add a comment.